labelbox/orm/db_object.py

```diff
@@ -244,3 +254,28 @@ def delete(self):
         a call to this you should not use this DB object anymore.
         """
         type(self).bulk_delete([self])
+
+
+def beta(fn):
```

Comment:
It was experimental before. I have been bouncing back and force on the naming. If you like experimental I'll switch it back because I have no idea which is best :)

Reply:
Switched back.